components/Drawer/__test__/drawer.spec.ts

```diff
@@ -84,13 +84,15 @@ describe('Test: KDrawer', () => {
 		expect(mockFn).toBeCalled();
 	});
 
-	test('slot: content', async () => {
-		const instance = new KDrawerContent({
-			target: host
+	test('slot: header', async () => {
```

Comment:
I still don't understand why you removed the slot related tests for content, can you explain?

Reply:
wait...i'll fix it

Reply:
I know why I blocked this... I've rolled back this content and added snap